boltdesignsystem/bolt
# DS-712 Remove twig whitespace added around tooltips

## Jira

https://pegadigitalit.atlassian.net/browse/DS-712

## Summary

Whitespace around the tooltip trigger was removed.

## Details

Whitespace was removed using CSS properties. `Spaceless` didn't want to work as mentioned in the Jira description.

## How to test

- Review code
- Check out this branch locally and see the new test page: `999-tests --> Tooltip Whitespace` (`/pattern-lab/?p=tests-tooltip-whitespace`)
- Check if there is no more whitespace around the tooltip trigger.
